lib/rubocop/cop/rspec/predicate_matcher.rb

```diff
@@ -110,7 +110,7 @@ def true?(to_symbol, matcher)
                      matcher.first_argument.true_type?
                    when :be_truthy, :a_truthy_value
                      true
-                   when :be_falsey, :be_falsy, :a_falsey_value, :a_falsy_value
+                   else # :be_falsey, :be_falsy, :a_falsey_value, :a_falsy_value
```

Comment:
Is this becase we don't want an "else" here?
Or because we don't want a separate test case for all four variants?
Might be we just add those four cases in specs and keep the cop intact?